client/src/components/Interview/subcomponents/questionAnswer.js

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import "../Interview.css"
+import UnispaceCircle from "../../../assets/UnispaceCircle.png";
+
+
+/**
+ * This is a question-answer dialogue box of the interview page.
+ * @param {string} question - question
+ * @param {string} answer - answer 
+ * @param {string} image - image of dialogue box
+ * @param {boolean} isHighlighted - whether it is introduction (green)
+ */
+
+const QuestionAnswer = ({ question, answer, image, isHighlighted }) => {
+    return (
+        <div className={isHighlighted ? 'questionDialogue highlighted' : 'questionDialogue'}>
+            <div className="flex questionTitle">
+                <img className="authorImg"
+                    style={isHighlighted ? { 'display': 'none' } : { 'display': 'flex', 'marginRight': '2%' }}
+                    src={UnispaceCircle}/>
+                <a> {question} </a>
```

Comment:
The intended design:
<img width="648" alt="image" src="https://github.com/TeamPathFinder/unispace/assets/66876469/59e12e4d-9624-49c8-9322-2f83cc88998c">

Actual design:
<img width="701" alt="image" src="https://github.com/TeamPathFinder/unispace/assets/66876469/6ce4b503-c5a7-4fde-9a8e-b9d763c0dd45">

Line height adjustments for the question text please ! A single line in a css file will do .. (i can commit but today i just realized that touching others' branches can cause big troubles so avoiding them :-(  )

+++ 

IDK how the design team wants this to be handled, but in some of their designs, instead of the unispace earth-like logo, they use `Q.` to start off questions. (tho imo your design seems more coherent)

Reply:
Hmm, theres like 4 separate designs and i don't know which one they want ㅠㅠㅠ

Reply:
Nice catch for the lineheight!